### DISCUSSION TITLE
HTTP Error: 406 Client Error: Not Acceptable for url

### DISCUSSION THREAD

**Discussion by athanell** [lassoan/SlicerDICOMwebBrowser#6] [Q&A] [answered]
Hello, 

I get a client error when connecting to an azure dicom webserver. I can see normally all the records through Slicer but when I try to download a series I get:

```
DownloadSelectedSeries: Error in getting response from DICOMweb server.
HTTP Error:
406 Client Error: Not Acceptable for url: https://xxx-xxx.azurewebsites.net/studies/1.2.826.0.1.xxxxxx.x.xxx.x.x.xxxxxxxxxxx.xxxx.xxxx/series/1.2.826.0.1.xxxxxx.x.x.x.x.xxxx.xxx.xxxxx.xxxxxx/instances/1.2.826.0.1.xxxx.x.xxx.x.x.8xxxxx.xxx.xxxxxxxx.xxxxx
```

Does this mean that there is an issue with the retrieval of the serie's DICOM instances (WADO)? the data itself (non conformant DICOMS)? something else?

Regards
Antonios

**Comment by lassoan** [comment, marked as answer]
The DICOMweb client requested something that the server could not fulfill, so it is hard to tell if the problem is on client or server side.

I've now [updated dicomweb-client to the latest version](https://github.com/lassoan/SlicerDICOMwebBrowser/commit/28a2806c62e7d1b3d9128c8e31810bbf9c19cc07) (a previous regression in the package required us to lock into <0.5 version). Please update your DICOMwebBrowser.py (or wait for the next nightly build and reinstall the extension tomorrow) and retry.

If you still have problems then report it to the [dicomweb-client](https://github.com/MGHComputationalPathology/dicomweb-client) issue tracker.

**Reply by athanell** [reply]
Thank you @lassoan. I've used the latest version but the issue still persists. I 'll report it to the proper thread.

**Reply by lassoan** [reply]
Sounds good. Add the link here to the submitted issue so that others that run into this in the future know where to follow up. Thanks!

**Reply by athanell** [reply]
The discussion continues in [dicomweb-client](https://github.com/MGHComputationalPathology/dicomweb-client/issues/46)
